package.json

```diff
@@ -66,8 +66,7 @@
     "@changesets/changelog-github": "0.2.6",
     "@changesets/cli": "2.9.2",
     "@microsoft/api-documenter": "7.8.21",
-    "@microsoft/api-extractor": "7.9.2",
-    "@types/mz": "2.7.1",
+    "@microsoft/api-extractor": "7.8.2-pr1796.0",
```

Comment:
This pulls in https://github.com/microsoft/rushstack/issues/1029, which makes this diff about 500 lines smaller (see the revert https://github.com/firebase/firebase-js-sdk/pull/3657/commits/609b2b3395b8b6988aed6996e45dc1f87a12e842)

Reply:
eh, the commit shows `445 additions and 477 deletions`, so the net gain seems to be 32 lines?

Reply:
FWIW, I created a new PR to merge just this: https://github.com/firebase/firebase-js-sdk/pull/3660